### DISCUSSION TITLE
How do I get the google-services.json file?

### DISCUSSION THREAD

**Discussion by Cosmopal** [nammayatri/nammayatri#359] [Q&A]
For Android, we need the google-services.json file for local compilation. How do we get this projects json?

**Comment by vikrantuk** [comment]
Hi @Cosmopal, you can create a new project in google firebase console, followed by creating new app in the project. Make sure to enter correct package name for your app.
Here is the detailed info on the same: [google-services.json](https://support.google.com/firebase/answer/7015592?hl=en#zippy=%2Cin-this-article)

**Comment by Cosmopal** [comment]
Ah okay. I was assuming there's a dev server. 

Do I need to do any additional setup in Firebase DBs, like creating dummy data? Or is there utilities to do that?

**Comment by vikrantuk** [comment]
Sorry, unfortunately I was not able to build frontend successfully on my
machine. So I'm not aware of this yet
